reviewresponse.cls

```diff
@@ -28,10 +28,11 @@
 \newcommand*{\themanuscript}{\@manuscript}
 
 %%% Colors
-\definecolor{colorcommentfg}{RGB}{0,63,87}
+\definecolor{colorconcluderesponsefg}{RGB}{0,63,87}
```

Comment:
It might be useful to either define two new separate colors for the new conclude comment box (for maximum customizability) or reuse the colors of the changes/general comment box.